src/ray/core_worker/core_worker.cc

```diff
@@ -1309,6 +1309,26 @@ Status CoreWorker::CreatePlacementGroup(
   return Status::OK();
 }
 
+Status CoreWorker::RemovePlacementGroup(const PlacementGroupID &placement_group_id) {
+  std::shared_ptr<std::promise<void>> promise = std::make_shared<std::promise<void>>();
```

Comment:
std::shared_ptr<std::promise< Status >> promise? and removal_status can be removed.

Reply:
Ah didn't know that syntax. Will do.